### PR TITLE
[UI] As a logged in user, I can see survey questions with Textfield and Slider

### PR DESCRIPTION
**Pull request by tyrro**
## What happened 👀
Closes https://github.com/tyrro/react-survey/issues/35
Closes https://github.com/tyrro/react-survey/issues/43

Textfield and Slider questions are implemented in this PR

## Insight 📝
 
This [stackoverflow](https://stackoverflow.com/questions/55644014/how-to-change-slider-thumb-color-using-react) answer helped me build an input range slider component without using any third party libraries!
 
The idea is to use `linear-gradient` background starting from the left,  then transitioning towards the right, with the currently active range percentage in a predefined dominant active color and the whole 100% of the background in a subdued inactive color. 
 
## Proof Of Work 📹

Textfield | Slider
--- | ---
image | image

## How to Test 🧪 

[This](https://react-survey-git-feature-gh-43-slider-questions-tyrro.vercel.app/surveys/eebff4e7a3ffd969a221/questions) URL contains both the Textfield and Slider component!

### CODE REVIEW

File: components/question/Textfield/index.jsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      }
+
+      return newSurveyQuestionWithAnswer;
+    });
+  };
+
+  return (
+    <div className="mt-16" data-test-id={textfieldTestIds.base}>
+      {textFieldData.map(textField => (
+        <Input
```
Comment (carryall) [inline components/question/Textfield/index.jsx:47]: Just saw that there's an `input_mask` attributes so not sure whether we need to work on validation or not 🤔  but as there's no design for validation so I guess no 😅 
Screen Shot 2565-05-19 at 11 16 22

File: components/question/RangeSlider/index.jsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      return [...newSurveyQuestionWithAnswer, surveyQuestionsWithAnswers];
+    });
+  };
+
+  const inputStyle = {
+    background: `linear-gradient(to right, ${ACTIVE_SLIDER_BACKGROUND} 0% ${progress}, ${INACTIVE_SLIDER_BACKGROUND} ${progress} 100%)`,
+  };
+
+  return (
+    <div className="h-14 flex items-center mt-16" data-test-id={rangeSliderTestIds.base}>
```
Comment (hoangmirs) [inline components/question/RangeSlider/index.jsx:41]: Should we move this CSS into `./index.module.scss` as well?

**Review by hoangmirs — APPROVED** [review]

File: components/question/RangeSlider/index.jsx (hunk 2 above)
Comment (carryall) [inline components/question/RangeSlider/index.jsx:41]: How about adding the `help_text` as `title` and/or `aria-label` for the slider?
Screen Shot 2565-05-19 at 11 33 06

**Review by carryall — COMMENTED** [review]
Looks like when displays multiple inputs and the style on focus is shifted to the next sibling 💭 

https://user-images.githubusercontent.com/1772999/169205196-6c9d64dd-6d06-403e-832a-72f15b15cbb0.mov

File: components/question/RangeSlider/index.jsx (hunk 2 above)
Comment (tyrro) [reply components/question/RangeSlider/index.jsx:41]: sure, done in 95ba02c628e6a3a0715a5157d5c16bfdbc2dfb17, Thanks!

**Review by carryall — APPROVED** [review]

**Comment by tyrro** [comment]
> Looks like when displays multiple inputs and the style on focus is shifted to the next sibling 💭
> 
As per our conversation on Slack, this one seems to be very hard to reproduce, I created a story here to track this issue: https://github.com/tyrro/react-survey/issues/22, Thank you!

File: components/question/RangeSlider/index.jsx (hunk 2 above)
Comment (tyrro) [reply components/question/RangeSlider/index.jsx]: I think it's a very good idea, I removed the wrapper div and moved these classes onto `./index.module.scss` in 3a7ab36a48d3b13c8c1c263372d878d4ed669872, Thank you!

**Pull request merged**